src/theory/arrays/theory_arrays.cpp

```diff
@@ -334,16 +334,18 @@ TrustNode TheoryArrays::ppRewrite(TNode term, std::vector<SkolemLemma>& lems)
       // select(store(a,i,v),j) = select(a,j)
       //    IF i != j
       if (term[0].getKind() == kind::STORE && ppDisequal(term[0][1], term[1])) {
-        ret = NodeBuilder<2>(kind::SELECT) << term[0][0] << term[1];
+        ret = NodeBuilder(kind::SELECT) << term[0][0] << term[1];
```

Comment:
It doesn't really make sense to use the `NodeBuilder` here. How about changing it to `nm->mkNode(kind::SELECT, term[0][0], term[1]);` ?